specs/boot_loader_specification.md

```diff
@@ -395,7 +395,8 @@ Images of this type have the advantage that all metadata and payload that makes
 up the boot entry is contained in a single PE file that can be signed
 cryptographically as one for the purpose of EFI SecureBoot.
 
-A valid unified kernel image in the `/EFI/Linux/` directory must contain two PE sections:
+A unified kernel image in the `/EFI/Linux/` directory may contain the following
+PE sections:
```

Comment:
I think it would make sense to drop the `.cmdline` section from the list below (and not make this a list at all then), since there may be many other sections. How about something something likes this

```
A unified kernel image in the `/EFI/Linux/` directory contains PE sections described in the [UKI Specification](unified_kernel_image.md).

Of these sections, the `.osrel` section is used to generate boot entries. 
The `PRETTY_NAME=` and `VERSION_ID=` fields in the embedded `os-release` file
are used the same as `title` and `version` in the Type #1 entries. The
`.cmdline` section is used instead of the `options` field. `linux` and `initrd`
fields are not necessary, and there is no counterpart for the `machine-id`
field.
```